data/tabular/peptides_soluble/meta.yaml

```diff
@@ -54,84 +54,84 @@ bibtex:
               year={2012},
               publisher={Wiley Online Library}
 templates:
-    - The sequence of {#amino acids|AAs!} {sequence#} {#shows|exhibits|demonstrates!} {soluble#no &NULL}{soluble__names__adjective} properties.
-    - The amino acid sequence {sequence#} {#shows|exhibits|displays!} {soluble#no &NULL}{soluble__names__adjective} properties.
-    - Based on the {sequence__description} {#representation |!}{sequence#}, the peptide has {soluble#no &NULL}{soluble__names__adjective} {#properties|characteristics|features!}.
-    - The {sequence__description} {sequence#} {#represents|is from!} a peptide that is {soluble#not &NULL}identified as {soluble__names__adjective}.
-    - The {#amino acid sequence |sequence of amino acids (AAs) |AA sequence|peptide with amino acid sequence!} {sequence#} is {soluble#not &NULL}{soluble__names__adjective}.
-    - |-
-      Task: Please classify a amino acid sequence based on the description.
-      Description: A amino acid sequence that is {soluble__names__adjective}.
-      {#amino acid sequence |sequence|AA sequence!}: {sequence#}
+    - The sequence of {#aminoacids|AAs!} {sequence#} {#shows|exhibits|demonstrates!} {soluble#no &NULL}{soluble__names__adjective} properties.
+    - The aminoacid sequence {sequence#} {#shows|exhibits|displays!} {soluble#no &NULL}{soluble__names__adjective} properties.
+    - Based on the {sequence__description} {#representation |!}{sequence#}, the aminoacid sequence has {soluble#no &NULL}{soluble__names__adjective} {#properties|characteristics|features!}.
+    - The {sequence__description} {sequence#} {#represents|is from!} an aminoacid sequence that is {soluble#not &NULL}identified as {soluble__names__adjective}.
+    - The {#aminoacid sequence |!}{sequence__description} {sequence#} is {soluble#not &NULL}{soluble__names__adjective}.
+    - |-
+      Task: Please classify a aminoacid sequence based on the description.
+      Description: A aminoacid sequence that is {soluble__names__adjective}.
+      {#aminoacid sequence |!}{sequence__description}: {sequence#}
       Constraint: Even if you are {#uncertain|not sure!}, you must pick either "True" or "False" without using any {#other|additional!} words.
       Result: {soluble#False&True}
     - |-
-      Task: Please classify a amino acid sequence based on the description.
-      Description: A amino acid sequence that is {soluble__names__adjective}.
-      {#amino acid sequence |sequence|AA sequence!}: {sequence#}
+      Task: Please classify a aminoacid sequence based on the description.
```

Comment:
I had change the spelling to amino acid before because https://www.merriam-webster.com/dictionary/amino%20acid https://dictionary.cambridge.org/us/dictionary/english-german/amino-acid